.travis.yml

```diff
@@ -52,9 +52,11 @@ script:
   - mvn -f pom-google-custom-suppression.xml clean checkstyle:checkstyle -Pgithub-maven-repo
   - grep "error" target/checkstyle-result.xml
   - mvn -f pom-google.xml clean checkstyle:checkstyle -Pgithub-maven-repo
+  - mvn checkstyle:checkstyle -Pgithub-maven-repo -Dcheckstyle.skip=true
   - cd ..
   - cd maven-ant-project
   - mvn verify
   - grep "error" target/checkstyle-result.xml
+  - mvn verify -Dcheckstyle.ant.skip=true
```

Comment:
ant-project (not to be confused with maven-ant-project) has no current skip property functionality coded.